### ISSUE TITLE
modifier if/else cop false positive

### ISSUE THREAD

**Issue by eqv** [rubocop/rubocop#231]
modifier if/else should not match if there is a multi line else / elsif etc

``` ruby
# encoding: utf-8

if 1 == 4
    puts 'bar'
elsif 2 == 3
    puts 'baz'
        puts 'fuz'
end
```

```
== test.rb ==
C:  5: Favor modifier if/unless usage when you have a single-line body. Another good alternative is the usage of control flow &&/||.

```
